### PR TITLE
F/validations-modelspecs

### PR DESCRIPTION
**Pull request by Gnwin**
## Blog App : f/validations-modelspecs

In this milestone, 
- ruby on rails was setup with postgres
- project schema was built
- models were created
- models were used to insert sample data with rails console
- validations and tests were written for models

All requirements have been met according to this document:
https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Gnwin, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow
✔️ All linters are passing

## Required 
- [ ] Kindly make sure all the tests are passing. Currently, 3 tests are not passing.
Screenshot 2022-07-24 at 8 04 43 AM

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/like_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.create(name: 'George', photo: 'www.sweetimage.com', bio: 'Super smart')
+  post = Post.create(user:, title: 'Title', text: 'sweet text!')
+
+  subject { Like.new(user:, post:) }
+
+  before { subject.save }
+end
```
Comment (smunozmo) [inline spec/models/like_spec.rb]: - [ ] You are almost there! However, there's still missing one test `CommentsCounter must be an integer greater than or equal to zero.`, as the last reviewer said, this test was not passing but it was deleted:

Screen Shot 2022-07-23 at 10 44 16 PM
 
Please add a Likes test as required by the activity.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Gnwin, 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @Gnwin 👍🏽, 
## Your tests are now passing 💯
### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
